Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -0,0 +1,98 @@
+description: Use Demisto server REST APIs
```

Comment:
The description looks weird..

![image](https://user-images.githubusercontent.com/5429033/33010655-f49228d8-cde3-11e7-92f5-
355b73f63615.png)

Reply:
@dorsha image didn't uploaded well

Reply:
@dorsha - can you re-upload so I can see what is the issue?

Reply:
Don't have the image, but I think the description should not start with the word "Use".
But not sure..

Reply:
ok. thought it is something with formatting/display. will rephrase.

Reply:
actually we do have a lot of integrations that start with a verb. so not sure too... 
@mayairv - what do you think?

![image](https://user-images.githubusercontent.com/1042046/33020293-6fcfae36-ce06-11e7-8c05-1c3de20c81f3.png)
![image](https://user-images.githubusercontent.com/1042046/33020295-795f9e8e-ce06-11e7-9ffc-4075df50dfed.png)
![image](https://user-images.githubusercontent.com/1042046/33020313-8f3c2704-ce06-11e7-9a90-6f3df6055969.png)